package.json

```diff
@@ -47,7 +47,7 @@
     "percy": "percy exec -- node snapshots.js",
     "icon-svgs-to-mixins": "node scripts/convert-svgs-to-icon-mixins.js icons"
   },
-  "version": "3.7.0",
+  "version": "3.7.1",
```

Comment:
I know it may be confusing a bit with multiple PRs, but as we haven't released 3.7 yet, we don't need to change the version.

Reply:
```suggestion
  "version": "3.7.0",
```